lua/bufferline/api.lua

```diff
@@ -394,11 +398,11 @@ function api.pick_buffer()
         if JumpMode.buffer_by_letter[letter] ~= nil then
           set_current_buf(JumpMode.buffer_by_letter[letter])
         else
-          notify("Couldn't find buffer", vim.log.levels.WARN, {title = 'barbar.nvim'})
+          notify("Couldn't find buffer", vim.log.levels.ERROR, {title = 'barbar.nvim'})
         end
       end
     else
-      notify("Invalid input", vim.log.levels.WARN, {title = 'barbar.nvim'})
+      notify("Invalid input", vim.log.levels.ERROR, {title = 'barbar.nvim'})
```

Comment:
These are errors, but we were reporting them as warnings. See the dedicated commit messahe for more details.